linera-core/src/client/mod.rs

```diff
@@ -1336,9 +1335,24 @@ where
             let client = self.clone();
             async move {
                 for certificate in certificates.into_values() {
```

Comment:
`ReceivedCertificatesFromValidator` should also contain `ConfirmedBlockCertificate`s.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/2832/commits/bc371b0e9309eb17084af0412e835e9e0e8b2589.